site/en/tutorials/images/transfer_learning.ipynb

```diff
@@ -69,7 +69,7 @@
         "id": "hRTa3Ee15WsJ"
```

Comment:
<p>It's a bit funny that the rescaling layer is applied to the dataset, but the data-augmentation isn't.</p><p>We create the <code>data_augmention</code> model, but do't use it here.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/2081/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
<p>Data augmentation layers have been applied, since:<code>data_augmentation = tf.keras.Sequential([tf.keras.layers.RandomFlip('horizontal'), tf.keras.layers.RandomRotation(0.2),])</code>  </p><p> </p><p>Let's just rewrite the <code>model</code> :</p><p> </p><pre class="ql-syntax" spellcheck="false">model = tf.keras.Sequential([
  data_augmentation,
  new_base_model,
  tf.keras.layers.Dropout(0.2),
  tf.keras.layers.Dense(num_classes)
]) 
</pre><p> </p><p>Earlier, I added two new sentences spelling out the layer names and descriptions + why we need this <code>data_augmentation</code> model. I also added a link to the KPL guide.</p><p> </p><pre class="ql-syntax" spellcheck="false">### Data augmentation

...

Create a data augmentation model using the <code>tf.keras.layers.RandomFlip</code> and <code>tf.keras.layers.RandomRotation</code> preprocessing layers. The former randomly flips images during training, while the latter—rotates them.

data_augmentation = ...
...

Later, you need to add <code>data_augmentation</code> as a layer inside your main Keras model before the training begins.

Note: These [preprocessing layers](https://www.tensorflow.org/guide/keras/preprocessing_layers) are active only during training when you call Keras <code>Model.fit</code>. They are inactive when the model is used in inference mode in <code>Model.evaluate</code> or <code>Model.fit</code>.

</pre>